egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -426,8 +439,49 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
     _train_dir="${data_feats}/${train_set}"
     _dev_dir="${data_feats}/${dev_set}"
     log "Stage 5: TTS Training: train_set=${_train_dir}, dev_set=${_dev_dir}"
-
+    
     _opts=
+    if [ -n "${teacher_model_path}" ] && echo "${train_config}" | grep -q "fastspeech"; then
+      # Setup feature and duration for fastspeech knowledge distillation training
```

Comment:
I'm not particular about it, but I used 4spaces indent in shell script, so please follow it.